rubrik_cdm/rubrik_cdm.py

```diff
@@ -112,6 +112,19 @@ def __init__(self, node_ip=None, username=None, password=None, api_token=None, e
                     self.password = password
                     self.log("Password: *******\n")
 
+                # Domain display name may be specified to force authentication to this domain
+                session_url = 'https://{}/api/internal/session'.format(self.node_ip)
+                ddn = domain_display_name if domain_display_name else os.environ.get('rubrik_cdm_domain')
```

Comment:
Can you break this out into something more verbose? Probably contrary to what you're used to but one of the main tenants of the SDK is having the code be as readable as possible to someone who is just getting started with programming and one-liners can be confusing for them